module.config.js.example

```diff
@@ -0,0 +1,31 @@
+module.exports = {
```

Comment:
Adding example `module.config.js` file to the repo as it has some helpful information on overriding modules for local development.